phenol-cli/src/main/java/org/monarchinitiative/phenol/cli/demo/PrecomputeScores.java

```diff
@@ -145,14 +143,14 @@ private void writeDistribution() {
 
     final int resolution = Math.min(1000, Math.max(100, numIterations / 100));
 
-    try (final ScoreDistributionWriter writer =
-        new TextFileScoreDistributionWriter(new File(this.outputScoreDistFile))) {
-      for (Entry<Integer, ScoreDistribution> e : scoreDistribution.entrySet()) {
-        writer.write(e.getKey(), e.getValue(), resolution);
-      }
-    } catch (IOException | PhenolException e) {
-      throw new RuntimeException("Problem writing to file", e);
-    }
+//    try (final ScoreDistributionWriter writer =
+//        new TextFileScoreDistributionWriter(new File(this.outputScoreDistFile))) {
+//      for (Entry<Integer, ScoreDistribution> e : scoreDistribution.entrySet()) {
+//        writer.write(e.getKey(), e.getValue(), resolution);
+//      }
+//    } catch (IOException | PhenolException e) {
+//      throw new RuntimeException("Problem writing to file", e);
+//    }
```

Comment:
Did you mean to leave this commented out?

Reply:
No, it should work OK now. Good catch, thanks! :slightly_smiling_face: